cli/src/test/resources/ide-projects/update/project/settings/ide.properties

```diff
@@ -0,0 +1,22 @@
+#********************************************************************************
+# This file contains project specific environment variables
+#********************************************************************************
+
+JAVA_VERSION=17.0.6
+MVN_VERSION=3.2.1
+ECLIPSE_VERSION=2023-03
+INTELLIJ_EDITION=ultimate
+
+IDE_TOOLS=java, mvn
+
+BAR=bar-${SOME}
+
+TEST_ARGS1=${TEST_ARGS1} settings1
+TEST_ARGS4=${TEST_ARGS4} settings4
+TEST_ARGS5=${TEST_ARGS5} settings5
+TEST_ARGS6=${TEST_ARGS6} settings6
+TEST_ARGS7=${TEST_ARGS7} settings7
+TEST_ARGS8=settings8
+TEST_ARGS9=settings9
+TEST_ARGSb=${TEST_ARGS10} settingsb ${TEST_ARGSa} ${TEST_ARGSb}

```

Comment:
Please avoid windows line endings.
Also what are these `TEST_ARGS*` variables used for? Also looks like pure copy&paste.